CONTRIBUTING.md

```diff
@@ -1,202 +1,151 @@
 
-> Note: The IR schema is modeled as a Fern Definition and you can see several
-> versions of them in the `./fern` folder.
+Once the CLI has been built, you can navigate to any `fern` folder and invoke it by running
 
-The Fern CLI should be able to run old generators so whenver we
-introduce a new IR version, we write a migration. In other words if you introduce IR V20, then
-you will have to write a migration from IR V20 -> IR V19 so that any generator
-that depends on a lower IR version can continue to be run from our CLI.
+```sh
+FERN_NO_VERSION_REDIRECTION=true node /<path to fern git repo>/packages/cli/cli/dist/dev/cli.cjsn <args>
+```
 
-### How to add a new IR Version?
+## Generators
 
-**Step 1: Define the new IR**
+All of Fern's generators live in a directory called [generators](./generators/). This directory contains generators for several languages such as
+[typescript](./generators/typescript/), [python](./generators/python/), [go](./generators/go).
 
-1. Create a new Fern Definition for the IR version `fern/ir-types-vXXX`.
-   Copy the latest IR Fern Definition as a starting point.
-2. Introduce any changes you want in the new IR Fern Definition.
-3. Generate a TypeScript SDK for the IR by running `fern generate --api ir-types-vXXX`
-4. Update all `package.json` files to use new `ir-sdk` npm version.
-   Run `yarn install`
-5. Run `pnpm compile`. You will see compile errors related to your schema changes.
+Some of the generators are written in the language they generate (i.e. Python is written in python, Go is written in Go, and Java is written in Java).
 
-**Step 2: Write a reverse migration**
+### Generator Testing
 
-In the `ir-migrations` package, introduce a new migration.
-You can copy the latest migration as a starting point.
+To test our generators we have built a CLI called seed.
 
-## Generator Testing (Seed CLI)
+Seed handles building the generators from source and running them against all of the
+[test definitions](./test-definitions/fern/) that are present in the repository. Generated code is then stored in a directory named
+[seed](./seed/).
 
-To test our generators we have built a CLI tool called seed.
+Each generator configures a `seed.yml`. For example, the TypeScript generator's configuration lives [here](./seed/ts-sdk/seed.yml).
 
-Seed handles building the generators from source and running them against all of the
-test definitions that are present in the repository. It also handles running scripts
-against the generated code to make sure that all the generated code compiles and
-works as intended.
+Seed also handles running scripts against the generated code to make sure that the generated code compiles and works
+as intended. For example, in the TypeScript generator seed runs `yarn install` and `yarn build` to compile the source code.
 
 To build seed, simply run
 
 ```sh
-yarn seed:build
+pnpm seed:build
```

Comment:
I use `seed:build` and `seed:test` since its faster and itll be even easier with turbo (seed test will depend on build). Cool to keep as is so we dont have to change after turbo migration?